infrastructure/resource_group_level/modules/webapps/dtfs-central-api.bicep

```diff
@@ -0,0 +1,105 @@
+param location string
```

Comment:
Ah - the rename has confused GitHub a bit. The change is I've pulled out the `site` creation and optional Application Insights resource into `webapp.bicep` and deleted all the overlapping code.

It has noticed correctly for `portal-api.bicep` below You might want to have a look at that for the sort of changes made to each file.